kubernetes/yamls/development/api-server/api-deployment.yml

```diff
@@ -18,13 +18,8 @@ spec:
           httpGet:
             path: /api/status.json
             port: 80
-          initialDelaySeconds: 30
-          timeoutSeconds: 3
-        readinessProbe:
-          httpGet:
-            path: /api/status.json
-            port: 80
-          initialDelaySeconds: 30
+          initialDelaySeconds: 90
+          periodSeconds: 300
           timeoutSeconds: 3
```

Comment:
I fixed this error by adding missing line here -> https://github.com/singhpratyush/loklak_server/compare/ee4ad16446d2f58fa0f47b537a8ddbcc82eb4c86...vibhcool:60a975a62fdf10ddca4a976da8a479620153df27